app/controllers/miq_ae_tools_controller.rb

```diff
@@ -395,7 +395,7 @@ def get_form_vars
       @resolve[:new][:target_class] = params[:target_class]
       klass = CustomButton.name_to_button_class(params[:target_class])
       unless klass.nil?
-        targets = klass.all
+        targets = klass
```

Comment:
Yeah, I like that idea better, as long as the `name_to_button_class` method isn't being called dynamically from somewhere else (which would be really odd, imo, but I've seen some odd things before so it wouldn't completely surprise me 😆 )